packages/cli/src/Generate.ts

```diff
@@ -126,9 +127,19 @@ ${bold('Examples')}
       '--postinstall': String,
       '--telemetry-information': String,
       '--allow-no-models': Boolean,
+      '--disallow-no-models': Boolean,
```

Comment:
Double negation. Maybe `--require-model` ?

Reply:
Or `--must-have-a-model` if "require" would be confusing.

Reply:
Made it `--require-models`, in plural. Singular makes sense logically but plural sounds a bit better to me, and is consistent with the opposite flag.